doc/tutorials/board_inspector_tool.rst

```diff
@@ -58,19 +59,25 @@ Board Inspector with the ``-h`` option yields the following usage message:
 .. code-block::
 
    usage: board_inspector.py [-h] [--out OUT] [--basic] [--loglevel LOGLEVEL]
-                [--check-device-status] board_name
+                             [--check-device-status]
+                             [--add-llc-cat <capacity_mask_length:int>,<clos_number:int>,<has_CDP:bool>]
+                             board_name
 
    positional arguments:
      board_name            the name of the board that runs the ACRN hypervisor
 
    optional arguments:
      -h, --help            show this help message and exit
      --out OUT             the name of board info file
-     --basic               do not extract advanced information such as ACPI namespace
-     --loglevel LOGLEVEL   choose log level, e.g. info, warning or error
+     --basic               do not extract advanced information such as ACPI
+                           namespace
+     --loglevel LOGLEVEL   choose log level, e.g. debug, info, warning, error or
+                           critical
      --check-device-status
-
                            filter out devices whose _STA object evaluates to 0
+     --add-llc-cat <capacity_mask_length:int>,<clos_number:int>,<has_CDP:bool>
```

Comment:
What are the valid values for `bool`?

Reply:
Y or N, I suspect?  We'd need to try or look at the code.

Reply:
I looked it up, it can be `true|false|y|n|yes|no` (not sure if this is case-sensitive) - see https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/board_inspector/board_inspector.py#L28